README.md

```diff
@@ -49,22 +51,23 @@ A WebSocket server requires a bind address, a port, and a coroutine to handle
 incoming connections. This example demonstrates an "echo server" that replies
 to each incoming message with an identical outgoing message.
 
-    import trio
-    from trio_websocket import WebSocketServer, ConnectionClosed
+```python
+import trio
+from trio_websocket import serve_websocket, ConnectionClosed
 
-    async def main():
-        server = WebSocketServer(echo_server, '127.0.0.1', 8000, ssl_context=None)
-        await server.listen()
+async def main():
+    await serve_websocket(echo_server, '127.0.0.1', 8000, ssl_context=None)
 
-    async def echo_server(websocket):
-        while True:
-            try:
-                message = await websocket.get_message()
-                await websocket.send_message(message)
-            except ConnectionClosed:
-                break
+async def echo_server(websocket):
```

Comment:
While we're here, I think it would read more logically if echo_server and main were swapped (i.e. in order for lowest level to highest).